### ISSUE TITLE
Freeze due to pause in pondering

### ISSUE THREAD

**Issue by hope366** [yzyray/lizzieyzy#22]
1. Set Max analyze to "10 seconds" and "100000 visits"
2. Start Yike live, execute synchronization, and close Yike live after execution. 
3. Start pondering 
4. Before the dialog to let you know that the contemplation was paused, display the browser all over (Lizzieyzy is on the back) and watch youtube for a while (about 10 minutes). 
5. When I put my browser in the taskbar and see what Lizzieyzy looks like, it's frozen and I have no choice but to kill it from the task manager. 

 Other than me, there are people who have encountered this problem. For that person, he set Max analyze to 10 minutes and left it for about 30 minutes before it freeze.
We haven't done much verification yet, so the conditions to ensure that this phenomenon is reproduced are still unclear. However, it doesn't seem to freeze for now under normal conditions without using the sync feature.

**Comment by yzyray** [comment]
You can check this option.When the issue happened again,see if there is anything in `LastErrorLogs` in lizzieyzy's folder.
![image](https://user-images.githubusercontent.com/49384584/141764371-ba38ecd8-112b-4239-8379-8a84c7474e33.png)

**Comment by hope366** [comment]
The contents of "LastErrorLogs_yzy2.4.5.txt" were blank and there was no record.
Also, a file called "LastConsoleLogs_yzy2.4.5.txt" was generated, and there was only a description of "java version: 11.0.12" here. 
By the way, I just downloaded a new version of Lizzieyzy, but neither the jar nor the exe file responded to double clicks ... the last 2.4.5 is working.

**Comment by yzyray** [comment]
Yes the exe and jar can not be excuted,I just updated the pre-release package ,It should be fine now.

**Comment by hope366** [comment]
I expected to get a new notification when the modified version was uploaded, but it doesn't seem to be such a specification.
I've just verified that 2.4.6 is working fine. Thank you.
I expected to get a new notification when the modified version was uploaded, but it doesn't seem to be such a specification.
I've just verified that 2.4.6 is working fine. Thank you.
Regarding the problem of GPU clock decrease, I thought that it was difficult because I could not see the comment that it was solved, but it seems that 2.4.6 is working well. Did you find any good solution?
